satpy/readers/li_base_nc.py

```diff
@@ -191,6 +191,7 @@
 import xarray as xr
 from pyproj import Proj
 
+from satpy.readers.fci_l1c_nc import _platform_name_translate
```

Comment:
Since we're now using this across modules, should we remove the `_` prefix?

Reply:
From my view yes. But will wait for other feedback.

Reply:
Without viewing the rest of this PR, I agree that something imported across modules should not have the `_` prefix.